server/honeybadger/plugins.py

```diff
@@ -10,8 +11,9 @@ def get_coords_from_google(aps):
         data['wifiAccessPoints'].append(ap.serialized_for_google)
     data_json = json.dumps(data)
     headers = {'Content-Type': 'application/json'}
-    request = urllib2.Request(url, data=data_json, headers=headers)
-    response = urllib2.urlopen(request)
+    request = urllib.request.Request(url, data="{}".encode(), headers=headers)
+    print(data_json)
```

Comment:
What' with the random print statement? Temporary debugging? If you need to see this info on a permanent basis, should it use the same logging function as everything else?

Reply:
Thanks for pointing this out. I believe this was a temporary debugging statement, and it can be removed. I don't see a reason that it needs to be logged separately, I just wanted to verify that the data was being sent correctly.

Reply:
Do a quick search for others that I may have missed and remove those as well.

Reply:
Perhaps I'm confusing myself, but I'm wondering why the data_json variable isn't being passed through the data value of urllib.request.Request. Strange that this module still seems to work, though perhaps it defaults to the caller's own machine. That should possibly be changed.

Reply:
A quick search through the python scripts for print statements shows the only other two being the database initialization and drop messages.

Reply:
Ok, cool. Those are fine.

Reply:
Line 14 definitely should be `request = urllib.request.Request(url, data=data_json.encode(), headers=headers)`. A new issue came up suddenly: The netsh command doesn't seem to be working like the iwlist command does. I might need to tweak the parser.

Reply:
I found using a state machine for these parsers is a more reliable approach.